dev/Dockerfile

```diff
@@ -18,88 +19,109 @@ RUN cargo chef prepare --recipe-path recipe.json
 # provides ARM binaries: https://github.com/LukeChannings/deno-arm64
 FROM lukechannings/deno:v${DENO_VERSION} AS deno-bin
 
-#
```

Comment:
this is a visual marker that we change build target, we can remove it if needed but everywhere below also